sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -418,7 +422,14 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
       .where("programs.programStatus = :programStatus", {
         programStatus: ProgramStatus.Approved,
       })
-      .andWhere("programs.isActive = true");
+      .andWhere("programs.isActive = true")
+      .andWhere(
+        new Brackets((qb) => {
+          qb.where("programs.effectiveEndDate is null").orWhere(
+            "programs.effectiveEndDate > CURRENT_DATE",
```

Comment:
`isSameOrAfterDate` used at line 379 will be inclusive, right? Should this condition be
`programs.effectiveEndDate >= CURRENT_DATE`?

Reply:
`isSameOfAfterDate` is used to check if `isExpired` is true;
In this case (getProgramsForLocation) we want to select the ones that are *NOT* expired (effectiveEndDate greater than the current date).